docs/kb/api-version-3/export.md

```diff
@@ -79,6 +79,46 @@ OR
 
 __NOTE:__ The range between created_after and created_before cannot exceed 1 year. When created_before is not specified, the current date is used to gauge the interval.
 
+## Prospect
+
+### Fields
+
+Select the prospect fields that need to be exported. [Following](../../object-field-references/#prospect) are the fields that are available for prospect. The value for `fields` must be an array of strings of the available fields. 
+
+### filter_by_updated_at
```

Comment:
Do we care about documenting `last_activity_at` filter?

Reply:
We did not implement the required deleted param. I think we should wait until we have the proper indexes on [last activity at](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B00000080n9aIAA/view). It is also not a general use case. @jpearson-sfdc  thoughts? I would tag Chris Cornett but I do not see a tag for him.